### ISSUE TITLE
"qs" dependency

### ISSUE THREAD

**Issue by bluntspoon** [jaystack/electron-ga#3]
I found I had to add a "qs" dependency to have it work in my Electron App, not sure if you are missing a dependency or if I might have done something wrong. 

Looks like you use it in your helper.ts
`import { stringify } from 'qs';`

Adding the below "qs" dendancy to my Electron app fixed the issue.
```
"main": "main.js",
"dependencies": {
	"electron-ga": "1.0.5",
	"qs": "6.5.1"
}
```

**Comment by HaNdTriX** [comment]
Looks like this issue can be closed since `qs` was added to the project in b06d06bc615bad677bbcc669405c9a9f812dc445.

Nevertheless, I recommend dropping this dependency in favor of the native [`URLSearchParams`](https://developer.mozilla.org/de/docs/Web/API/URLSearchParams)

**Comment by bluntspoon** [comment]
closed since qs was added to the project
